From 75ddaf5a854fcdccacfc722660758506e4ff4eca Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Tue, 27 Nov 2012 10:22:48 -0800
Subject: [PATCH] mtd: xilinx_nandps: Remove redundant callback assignments

The platform_driver's 'suspend' and 'resume' members were assigned NULL.
Thus, those statements can be removed.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
Acked-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/mtd/nand/xilinx_nandps.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/drivers/mtd/nand/xilinx_nandps.c b/drivers/mtd/nand/xilinx_nandps.c
index c408ff8..3108971 100644
--- a/drivers/mtd/nand/xilinx_nandps.c
+++ b/drivers/mtd/nand/xilinx_nandps.c
@@ -1294,8 +1294,6 @@ static int __devexit xnandps_remove(struct platform_device *pdev)
 static struct platform_driver xnandps_driver = {
 	.probe		= xnandps_probe,
 	.remove		= __devexit_p(xnandps_remove),
-	.suspend	= NULL,
-	.resume		= NULL,
 	.driver		= {
 		.name	= XNANDPS_DRIVER_NAME,
 		.owner	= THIS_MODULE,
-- 
1.7.1

